ota-plus-web/app/reactapp/src/components/dashboard/new/DashboardStepper/styled.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { Steps } from 'antd';
+
+const { Step: AntStep } = Steps;
+
+const CardsWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 52px 10px 10px 10px;
+  border-radius: 1px;
+`;
+
+const Step = styled(AntStep)`
+  margin-right: 0 !important;
+  .ant-steps-item-icon {
+    margin-right: 0;
+    width: 48px;
+    height: 48px;
+    border: 1px solid #12E4D6;
+    background-color: #1B222C;
+    span {
+      font-size: 1.38em;
+      font-weight: 400;
+      top: 8px;
+      color: rgba(255,255,255,0.6) !important;
+    }
+  }
+`;
+
+const StepperWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: #3F454D;
```

Comment:
Why not add the colors to styleConstants?

Reply:
I suggest we align all the palette colors once the rework is finished, otherwise it'll get messy really quick.